### PR TITLE
bsunanda:Phase2-ft14 Correct FastTimer Reco Geometry and add a testing function

### PR DESCRIPTION
**Pull request by bsunanda**

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalGeometry/test/python/testFastTimeGeometry_cfg.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    debugModules = cms.untracked.vstring('*'),
+    cout = cms.untracked.PSet(
+        threshold = cms.untracked.string('DEBUG'),
+        default = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        HGCalGeom = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        )
+    ),
```
Comment (ianna) [inline Geometry/HGCalGeometry/test/python/testFastTimeGeometry_cfg.py]: @bsunanda - I guess, this comma is a typo?

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ianna** [comment]
+1

**Comment by lgray** [comment]
@davidlange6 Could you please merge this one soon? I am trying to finalize RecHits and TrackingGeometry for the timing layer.

**Comment by davidlange6** [comment]
+1

**Pull request merged**